src/EditModel/EditDataEntryForm/processFormData.test.js

```diff
@@ -0,0 +1,34 @@
+import { generateHtmlForId, processFormData } from './processFormData';
```

Comment:
I added some basic tests for `processFormData` after extracting it to a separate function